singleapplication_p.cpp

```diff
@@ -301,7 +301,13 @@ bool SingleApplicationPrivate::writeConfirmedMessage (int msecs, const QByteArra
         return false;
 
     // Frame 2: The message
-    return writeConfirmedFrame( static_cast<int>(msecs - time.elapsed()), msg );
+    const bool result = writeConfirmedFrame( static_cast<int>(msecs - time.elapsed()), msg );
+
+    // Block if needed
+    if (socket && (sendMode == SingleApplication::BlockUntilPrimaryExit))
```

Comment:
We usually omit extra brackets:
```suggestion
    if (socket && sendMode == SingleApplication::BlockUntilPrimaryExit)
```